internal/eval/eval.go

```diff
@@ -3439,3 +3440,30 @@ func evalJSONNUMINCRBY(args []string, store *dstore.Store) []byte {
 	obj.Value = jsonData
 	return clientio.Encode(resultString, false)
 }
+
+func evalTYPE(args []string, store *dstore.Store) []byte {
+	if len(args) != 1 {
+		return diceerrors.NewErrArity("TYPE")
+	}
+	key := args[0]
+	obj := store.Get(key)
+	if obj == nil {
+		return clientio.Encode("none", false)
+	}
+
+	var typeStr string
+	switch obj.TypeEncoding & ObjTypeMask {
+	case object.ObjTypeString:
+		typeStr = "string"
+	case object.ObjTypeByteList:
+		typeStr = "list"
+	case object.ObjTypeSet:
+		typeStr = "set"
+	case object.ObjTypeHashMap:
+		typeStr = "hash"
+	default:
+		typeStr = "non-supported type"
```

Comment:
Also, I checked the code for data created using setbit commands. It should return "string" but dice returns "non-supported type". Please find evidence below:=
![image](https://github.com/user-attachments/assets/d5b0a43c-3cbd-4ccc-94a1-6a26a6eef422)
![image](https://github.com/user-attachments/assets/3f617308-84c3-40be-9a2f-1b0e853b27ce)